### PR TITLE
chore: remove cta

### PR DESCRIPTION
**Pull request by bennypowers**
:axe:

### CODE REVIEW

**Comment by bennypowers** [comment]
The big whooptie here is that CTA will no longer be available for the docs site...

**Comment by bennypowers** [comment]
I've added some basic CTA styles to the docs site's `main.css`, to replace the now-defunct `<pfe-cta>` element. It's not perfect (theme colours don't work, the context colours aren't 1-to-1), but I think it's Good Enough:tm:, since afaict everything's basically legible. I'd also rather not spend that much more time on this, considering the overall goals of 1:1

File: core/pfe-styles/demo/colors.html [hunk 1]
```diff
@@ -125,15 +124,15 @@ <h2 id="colors">Color Palette</h2>
...
+            </pfe-btton>
+            <pfe-btton variant="primary" color-palette="complement" class="themes dark">
               <button name="dark">Dark theme</button>
-            </pfe-cta>
-            <pfe-cta priority="primary" color-palette="base" class="themes saturated">
+            </pfe-btton>
+            <pfe-btton variant="primary" color-palette="base" class="themes saturated">
               <button name="saturated">Saturated theme</button>
-            </pfe-cta>
+            </pfe-btton>
```
Comment (zeroedin) [inline core/pfe-styles/demo/colors.html:135]: Typo fix

I'm also not sure this HTML page is displayed anywhere.  I couldn't find anything that linked to https://patternflyelements.org/core/styles/demo/colors.  Maybe able to delete it moving forward? 

```Suggestion
            <pfe-button variant="secondary" class="themes light">
              <button name="light">Light theme</button>
            </pfe-button>
            <pfe-button variant="primary" color-palette="complement" class="themes dark">
              <button name="dark">Dark theme</button>
            </pfe-button>
            <pfe-button variant="primary" color-palette="base" class="themes saturated">
              <button name="saturated">Saturated theme</button>
            </pfe-button>
```

File: elements/pfe-card/demo/pfe-card.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import '@patternfly/pfe-band';
 import '@patternfly/pfe-card';
-import '@patternfly/pfe-cta';
+import '@patternfly/pfe-button';
```
Comment (zeroedin) [inline elements/pfe-card/demo/pfe-card.js:3]: pfe-card demo doesn't appear to use `pfe-button`

```Suggestion
```

**Review by zeroedin — DISMISSED** [review]
A couple of small requested changes otherwise good to go.

File: core/pfe-styles/demo/colors.html (hunk 1 above)
Comment (bennypowers) [reply core/pfe-styles/demo/colors.html:135]: I'm happy deleting the pfe-styles package altogether, i'm just not sure who's using it.

**Comment by heyMP** [comment]
@zeroedin your review as addressed since you'll be traveling :)

**Review by heyMP — APPROVED** [review]
👍

**Pull request merged**